job-dsls/jobs/submarine.groovy

```diff
@@ -0,0 +1,291 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def javadk=Constants.JDK_VERSION
+def javaToolEnv="KIE_JDK1_8"
+def mvnToolEnv="KIE_MAVEN_3_5_4"
+def mvnVersion="kie-maven-3.5.4"
```

Comment:
would it be possible to get maven 3.6.0 as it is needed for quarkus?

Reply:
I don't think we have this yet on our machines . have to speak to people

Reply:
ok, let's make this in and then fix the version of maven .. this version of maven is only needed for quarks based examples so we can leave without them for now.

Reply:
but I think @mbiarnes already prepared everything for version 3.6.0. so we don't need to wait.